models/articles.models.js

```diff
@@ -6,4 +6,15 @@ exports.fetchArticleById = async (id) => {
         return Promise.reject({ status: 404, msg: "Article not found" })
     }
     return rows[0]
-}
\ No newline at end of file
+}
+
+exports.fetchArticle = async () => {
+    const { rows } = await db.query(`
+            SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comment_id) AS comment_count
+            FROM articles
+            LEFT JOIN comments ON articles.article_id = comments.article_id
+            GROUP BY articles.article_id
+            ORDER BY articles.created_at DESC
+    `)
+    return rows
+}
```

Comment:
Great work on this SQL query! 👏 As mentioned in my previous comment, you can have a look into how to get the comment count come out as a number if you'd like 😄